commit 077fdce94e1a8305ead51d67441de9ee2295fac3
Author: PJBrs <pjbrs@floorenpj.xs4all.nl>
Date:   Fri Feb 21 08:58:30 2020 +0100

    ARM: dts: sunxi: h2-plus-bananapi-m2-zero: Add HDMI audio out
    
    Add sound over hdmi to BananaPi M2 Zero.

diff --git a/arch/arm/boot/dts/sun8i-h2-plus-bananapi-m2-zero.dts b/arch/arm/boot/dts/sun8i-h2-plus-bananapi-m2-zero.dts
index 9051a1097258..0b702168981c 100644
--- a/arch/arm/boot/dts/sun8i-h2-plus-bananapi-m2-zero.dts
+++ b/arch/arm/boot/dts/sun8i-h2-plus-bananapi-m2-zero.dts
@@ -165,6 +165,10 @@
 	};
 };
 
+&mixer0 {
+	status = "okay";
+};
+
 &ohci0 {
 	status = "okay";
 };
@@ -217,3 +221,11 @@
 	phy-mode = "mii";
 	allwinner,leds-active-low;
 };
+
+&i2s2 {
+	status = "okay";
+};
+
+&sound_hdmi {
+	status = "okay";
+};
